packages/mcp/src/configuration.ts

```diff
@@ -141,6 +141,84 @@ To fix this you have three different options:
     return connectedResources;
   }
 
+  /**
+   * Get all resource templates from connected MCP servers
+   * @returns A record of server names to their resource templates
+   */
+  public async listResourceTemplates(): Promise<Record<string, ResourceTemplate[]>> {
```

Comment:
these should be `MCPClient.resources.templates()`, `MCPClient.resources.read()`, etc. Once we add everything else (prompts, roots, etc) there will be too many methods